sql/augments.sql

```diff
@@ -394,31 +395,30 @@ INSERT INTO `augments` VALUES (318,0,0,0,0,0);
 INSERT INTO `augments` VALUES (319,0,0,0,0,0);
 -- End unused block
 
-INSERT INTO `augments` VALUES (320,0,357,-1,0,0); -- Blood Pact ability delay -1
-INSERT INTO `augments` VALUES (321,0,346,-1,0,0); -- Avatar perpetuation cost -1
-INSERT INTO `augments` VALUES (322,0,455,-1,0,0); -- Song spellcasting time -1%
-INSERT INTO `augments` VALUES (323,0,0,0,0,0); -- Cure spellcasting time -1%
+INSERT INTO `augments` VALUES (320,0,357,1,0,0); -- Blood Pact ability delay -1
```

Comment:
sidenote to team and all contributors: I hate how we do that.

"is it positive or negative?"
"yes"

Reply:
2 ways to solve this:
1. Turn them negative when they are gonna be deducting.
2. Change name to "Blood Pact ability delay REDUCTION" or something of the short, to make it clear the value will be being deduced.